src/transformFontWeight.ts

```diff
@@ -47,21 +49,31 @@ export function transformFontWeight(token: DesignToken): DesignToken['value'] {
   const type = token.$type ?? token.type;
   if (val === undefined) return undefined;
 
-  const transformWeight = (weight: number | string) => {
-    const match = `${weight}`.match(fontWeightReg);
+  const transformWeight = (fontWeight: number | string) => {
+    const match = `${fontWeight}`.match(fontWeightReg);
+    if (!match?.groups) {
+      return fontWeight;
+    }
 
     let mapped;
-    if (match?.groups?.weight) {
-      mapped =
-        fontWeightMap[
-          match?.groups?.weight.replace(/\s/g, '').toLowerCase() as keyof typeof fontWeightMap
-        ];
-      if (match.groups.style) {
-        mapped = `${mapped} ${match.groups.style.toLowerCase()}`;
-      }
+    // lowercase to make the following code case insensitive
+    const weight = match.groups.weight.toLowerCase() as keyof typeof fontWeightMap;
+    const style = match.groups.style ? match.groups.style.toLowerCase() : '';
```

Comment:
```suggestion
    const style = match.groups.style?.toLowerCase() ?? '';
```
I realize now that this would be cleaner if possible, double check if that's valid though